modules/serverless-logic-overview-error-handling.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+// * about/serverless-logic-overview.adoc
+
+
+:_content-type: CONCEPT
+[id="serverless-logic-overview-error-handling_{context}"]
+= Error handling
+
+Openshift Serverless Logic allows you to define `explicit` error handling, meaning you can define what should happen in case of errors inside your workflow model rather than some generic error handling entity. It enables you to handle the errors that might happen during the interactions between the workflow and external systems. When an error occurs, it changes the regular workflow sequence. In such cases, a workflow state transitions to an alternative state that can potentially handle the error, instead of transitioning to the predefined state.
+
+Each workflow state can define error handling, which is related only to errors that may arise during its execution. Error handling defined in one state cannot be used to handle errors that happened during execution of another state during workflow execution.
+
+Unknown errors that may arise during workflow state execution that are not explicitly handled within the workflow definition should be reported by runtime implementations and halt workflow execution.
+
+== Error definition 
```

Comment:
I'm only going to mark this once, but all headings need an anchor ID, and headings in a module also require the "context" variable: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings

Reply:
Okay! I will include this now.

Reply:
I have added the anchor IDs and the "context" variables.